src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -48,11 +69,38 @@
 		public static EnableableModifier rightWheelAccelerationCap;
 		public static EnableableModifier leftWheelAccelerationCap;
 	}
+
+	public static class HumanInput {
+		public static class Driver {
+			public static CustomXbox xbox;
+		}
+
+		public static class Operator {
+			public static CustomJoystick leftStick;
+			public static CustomJoystick rightStick;
+		}
+	}
 	/**
 	 * The static initializer runs exactly once and ensures that
 	 * the variables are properly initialized.
 	 */
 	static {
+		Motor elbowMotorA = new Motor("elbowMotorA", new CANTalonSRX(Port.CANMotor.elbowMotorA)); // use TalonSRX when we have it
```

Comment:
Is there anything else that distinguishes motor "A" & "B", like left and right?

Reply:
No. A and B are on the same joint running in opposite directions and are both controlled by the SAME joystick.